packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -2160,7 +2238,7 @@ function validateRootField(tree: ISharedTreeBranch, expected: Value[]): void {
 	let hasNode = readCursor.firstNode();
 	for (const value of expected) {
 		assert(hasNode);
-		assert.equal(readCursor.value, value);
+		// assert.equal(readCursor.value, value);
```

Comment:
Intentional change?

Reply:
Sorry, I'm actually still working on the branch, so it's still not quite ready for a review yet!